### DISCUSSION TITLE
Proxy setting

### DISCUSSION THREAD

**Discussion by yumccarthy** [sql-bi/Bravo#641] [General]
Proxy setting doesn't work if I select option - "System". 
Only works when I use option - "Custom" to fill in proxy server address.
May I know why "system" doesn't work? Anyone has same problem?

And compared with Power BI desktop and Power BI report builder, they both can enable default proxy by adding registry key. Is it also available for Bravo?

**Comment by albertospelta** [comment]
Hi @yumccarthy, configuring proxy settings via registry key is not currently supported in Bravo but it is an enhancement that we could definitely evaluate and add in a future release. I just added a new task to https://github.com/sql-bi/Bravo/issues/410 to track this request.

> Proxy setting doesn't work if I select option - "System".

Do you get any error message when using the "System" proxy option?

> Only works when I use option - "Custom" to fill in proxy server address.

Would you mind sharing an example of the address you use as custom proxy address?

Thanks!
